recipes/xlandsat/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "xlandsat" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/xlandsat-{{ version }}.tar.gz
+  sha256: fab7e2c42cb9f85feafaf5e41dbcf7b66bb7079618dec2b6b9e9eb2fc6d3d92e
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
```

From the `pyproject.toml`:
```suggestion
    - python >=3.7
    - pip
    - setuptools
    - setuptools_scm
    - tomli
```

`setuptools_scm` is the important one since while this build was passing, it was coming up with version 0.0.0.

---

😢 I thought it wouldn't be needed since the package is coming from PyPI and should have the version file in there.

---

I can see it in the sdist, but it also regenerated the file when I ran install (though pip downloaded setuptools_scm for me), so something might not quite be setup how you want.